perf/stability/external-traffic/README.md

```diff
@@ -0,0 +1,5 @@
+# External Traffic
+
+This tests external traffic without using `ServiceEntry`s.
+
+Instead, `global.outboundTrafficPolicy.mode=ALLOW_ANY` is set and a Sidecar resource is created.
```

Comment:
Please add  Checks for
1.  Ensure that this flow works `externaltraffic Deployment` --> (ALLOW_ANY) --> ingress gateway --> fortioServer
2. Ensure that an unrelated serviceEntry in the `externaltraffic` does not interfere